enmeshed/main.tf

```diff
@@ -0,0 +1,258 @@
+locals {
+  mongodb_uri = "mongodb://mongo:pass@enmeshed-db:27017/enmeshed-db"
+}
+
+resource "kubernetes_deployment" "enmeshed_deployment" {
+  metadata {
+    name      = "enmeshed-deployment"
+    namespace = var.namespace
+  }
+
+  spec {
+    selector {
+      match_labels = {
+        app = "enmeshed-app"
+      }
+    }
+
+    template {
+      metadata {
+        namespace = var.namespace
+        labels = {
+          app = "enmeshed-app"
+        }
+      }
+      spec {
+        container {
+          image = "ghcr.io/nmshd/connector:${var.image_tags.enmeshed}"
+          name  = "connector"
+
+          env {
+            name  = "CUSTOM_CONFIG_LOCATION"
+            value = "/config.json"
```

Comment:
See https://github.com/serlo/infrastructure-modules-shared/blob/main/hydra/main.tf#L14 for how to create `config.json` dynamically via terraform.

Reply:
Resolved in 8f1cbd3 (at the moment in branch enmeshed-refactored)